Vision/README.md

```diff
@@ -11,34 +9,34 @@ Our vision is for a World Wide Web that is more inclusive, and more respectful o
 
 We will improve the fundamental integrity of the Web platform. The Web will not only grow in scope and importance in our lives; it will grow in respect for its users, grow in the trust of its users, grow in its inclusion of all humanity as its users.
 
-Our new vision is built on these twin pillars:
-
-* Our existing pillar, on which we built the web: the enormous potential of technological change.
-* Our new pillar for the years ahead: recognition and embodiment of fundamental values into the architecture of the web.
-
-# Our Values for the world-wide web
-
-These are the core values of the Web:
+# Vision for the World-Wide Web
```

Comment:
I believe it's clearer to call this what it is - the Vision of the Web.

Reply:
Replicating comments from https://github.com/w3c/AB-public/pull/28#discussion_r1048814564:

(thumbs-up from mchampion)

@[dwsinger](https://github.com/dwsinger) dwsinger [1 hour ago](https://github.com/w3c/AB-public/pull/28#discussion_r1049048172)
Sorry, no. Our vision for the web may be that these values are fundamental, but a vision for the web itself woudl say something about how it's going to change. These are underlying principles, not a vision of its actual development. That would be more like "we see blockchain technologies fading away again; more monetary management (e.g. including micro-payments)" and so on